### PR TITLE
Add Windows support for toolbox image

### LINKED ISSUE #26: e2e: Windows e2e tests failed because toolbox has no windows versions
**Describe the bug**

https://github.com/antrea-io/antrea/pull/5880 changed to use toolbox to test ping as it's packaged with a version supporting `-M do` flag. However, the test can be ran on windows platform and toolbox doesn't have a windows version.

Due to this, currently "antrea-windows-containerd-e2e" would always fail with the following tests:
```
    --- FAIL: TestConnectivity/testPodConnectivityOnSameNode (90.04s)
    --- FAIL: TestConnectivity/testPodConnectivityDifferentNodes (90.04s)
    --- FAIL: TestConnectivity/testPingLargeMTU (90.01s)
```
We need to either change the image when the test runs on Windows, or add a Windows version of toolbox.

### PR DESCRIPTION
**Pull request by antoninbas**
We add Windows images to the antrea/toolbox manifest (for both the 1809 and ltsc2022 Windows nanoserver versions).

The Windows image is based directly on K8s pause image. It includes wincat, pause, and of course everything included by default in the nanoserver image (such as ping).

In the future, we may want to install additional utilities in the Windows image.

Because we share the same versioning between Linux and Windows, the toolbox version is bumped up to 1.3.

Fixes #26

### CODE REVIEW

File: .github/workflows/docker_update_toolbox.yml [hunk 1]
```diff
@@ -13,23 +13,42 @@ on:
...
         rev=$(git rev-list --count ${version_update_sha}..HEAD -- .)
-        echo "version=${version}" >> $GITHUB_ENV
-        echo "rev=${rev}" >> $GITHUB_ENV
-        echo "docker_tag=${version}-${rev}" >> $GITHUB_ENV
+        echo "version=${version}" >> $GITHUB_OUTPUT
+        echo "rev=${rev}" >> $GITHUB_OUTPUT
+        echo "docker_tag=${version}-${rev}" >> $GITHUB_OUTPUT
+    outputs:
+      version: ${{ steps.get-outputs.outputs.version }}
+      rev: ${{ steps.get-outputs.outputs.rev }}
```
Comment (tnqn) [inline .github/workflows/docker_update_toolbox.yml:39]: this seems not used

File: images/toolbox/Dockerfile-windows [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# See https://kubernetes.io/docs/concepts/windows/intro/#pause-container
+# Unlike the image from registry.k8s.io, there are also different named tags for different OS
+# versions, so we have more control.
+FROM mcr.microsoft.com/oss/kubernetes/pause:3.9-windows-${WINDOWS_VERSION}-amd64
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="A Docker image with useful tools (for networking) based on the K8s pause image"
+
+# We do not add any tools to the image at the moment.
+# The image comes with pause + wincat + anything included by default in the nanoserver base image.
```
Comment (tnqn) [inline images/toolbox/Dockerfile-windows:27]: Does it have ping?

**Review by tnqn — APPROVED** [review]
LGTM

File: images/toolbox/Dockerfile-windows (hunk 2 above)
Comment (antoninbas) [reply images/toolbox/Dockerfile-windows:27]: Yes, I checked. It's included by default in the nanoserver image

File: .github/workflows/docker_update_toolbox.yml (hunk 1 above)
Comment (antoninbas) [reply .github/workflows/docker_update_toolbox.yml:39]: yes, it's true but I didn't feel the need to remove it

**Pull request merged**